docs/administration.mesh-policies.md

```diff
@@ -3,25 +3,26 @@ id: administration.mesh-policies
 title: meshPolicies
 ---
 
-meshPolicies are described in detail [here](meshcloud.policies.md). They can be set up between meshObjects, i.e., meshCustomer, meshProject,
+meshPolicies are described in detail [here](meshcloud.policies.md). They can be set up between meshSubjects, i.e., meshCustomer, meshProject,
```

Comment:
I'd rather ditc the `i.e.` part and just list all meshSubjects that we have in a list:

```
The following meshSubjects are available:
- meshCustomer
- [etc...]
```

Reply:
Fxied.